Circular import in blueprints tests

https://github.com/terrapower/armi/blob/2571216afadbd0b773555f3dfc3d77b66840fa8f/armi/reactor/blueprints/tests/test_gridBlueprints.py#L26

While reviewing a PR, I (well, vscode's debugger) found this circular import. 

```
=================================== ERRORS ====================================
_ ERROR collecting framework/armi/reactor/blueprints/tests/test_gridBlueprints.py _
ImportError while importing test module 'c:\Users\aalberti\codes\nala\framework\armi\reactor\blueprints\tests\test_gridBlueprints.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
C:\tools\Python\Python39\lib\importlib\__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
framework\armi\reactor\blueprints\tests\test_gridBlueprints.py:26: in <module>
    from armi.reactor.blueprints.tests.test_blockBlueprints import FULL_BP, FULL_BP_GRID
framework\armi\reactor\blueprints\tests\test_blockBlueprints.py:21: in <module>
    from armi.reactor.tests import test_blocks
framework\armi\reactor\tests\test_blocks.py:26: in <module>
    from armi.reactor.blueprints.tests.test_blockBlueprints import FULL_BP
E   ImportError: cannot import name 'FULL_BP' from partially initialized module 'armi.reactor.blueprints.tests.test_blockBlueprints' (most likely due to a circular import) (c:\Users\aalberti\codes\nala\framework\armi\reactor\blueprints\tests\test_blockBlueprints.py)
=========================== short test summary info ===========================
ERROR framework/armi/reactor/blueprints/tests/test_gridBlueprints.py
!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
==================== no tests collected, 1 error in 0.87s =====================
```

--- a/armi/reactor/blueprints/tests/test_gridBlueprints.py
+++ b/armi/reactor/blueprints/tests/test_gridBlueprints.py
@@ -23,7 +23,6 @@ if not isConfigured():
 from armi.reactor import systemLayoutInput
 from armi.reactor.blueprints import Blueprints
 from armi.reactor.blueprints.gridBlueprint import Grids, saveToStream
-from armi.reactor.blueprints.tests.test_blockBlueprints import FULL_BP, FULL_BP_GRID
 from armi.utils.directoryChangers import TemporaryDirectoryChanger
 
 
@@ -401,6 +400,8 @@ class TestGridBlueprintsSection(unittest.TestCase):
             :id: T_ARMI_BP_GRID0
             :tests: R_ARMI_BP_GRID
         """
+        from armi.reactor.blueprints.tests.test_blockBlueprints import FULL_BP
+
         # Cartesian full, even/odd hybrid
         gridDesign4 = self.grids["sfp even"]
         _grid = gridDesign4.construct()
@@ -432,6 +433,8 @@ class TestGridBlueprintsSection(unittest.TestCase):
         self.assertTrue(os.path.exists(filePath))
 
     def test_simpleReadNoLatticeMap(self):
+        from armi.reactor.blueprints.tests.test_blockBlueprints import FULL_BP_GRID
+
         # Cartesian full, even/odd hybrid
         gridDesign4 = self.grids["sfp even"]
         _grid = gridDesign4.construct()

